course-recommendation.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: page
+title: "通识水课推荐"
+permalink: /course-recommendation
+---
+
+<!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-4DT3EE5Z3Q"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+
+  gtag('config', 'G-4DT3EE5Z3Q');
+</script>
+<script data-ad-client="ca-pub-3457849876540251" async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
+
+<div id="google_translate_element" style="float:right"></div>
+<script>
+function googleTranslateElementInit() {
+  new google.translate.TranslateElement({pageLanguage: 'zh'}, 'google_translate_element');
+}
+</script>
+<script async src="https://translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
+
+<div style="padding-bottom: 6px">
+<a href="http://www.mingcns.org">返回文档索引主页</a>
+</div>
+
+# 新文档模板
+
+{:.no_toc}
+
+<div align="right">
+本指南非官方文件，详见<a href="#版权与免责协议">版权与免责协议</a><br>
+(Version: 2000-01-01)
+</div><br>
+
+## 目录
+
+{:.no_toc}
+
+- 目录
+  {:toc}
+
+## 通识水课推荐：
+
+<table>
+  <tr>
+    <th>课号</th>
+    <th>课程名称</th>
+    <th>满足标签</th>
+  </tr>
+  <tr>
+    <td>JOUR 1501 *Online</td>
+    <td>Digital Games and Society</td>
+    <td>Arts/Humanities & Technology and Society</td>
+  </tr>
+  <tr>
+    <td>MUS 1013 *Oneline</td>
+    <td>Rock I: The Historical Origins and Development of Rock Music to 1970</td>
+    <td>Arts/Humanities Diversity & Soc. Justice in the U.S</td>
+  </tr>
+  <tr>
+    <td>AMST 2011</td>
+    <td>The United States since September 11</td>
+    <td>Civic Life & Ethics Historical Perspectives</td>
+  </tr>
+  <tr>
+    <td>AMST 2031</td>
+    <td>Chasing the American Dream: Economic Opportunity & Inequality in the U.S.</td>
+    <td>Diversity and Soc. Justice in the U.S. & Historical Perspectives</td>
+  </tr>
+  <tr>
+    <td>HSCI 1011</td>
+    <td>Digital World</td>
+    <td>Historical Perspectives & Technology and Society</td>
+  </tr>
+  <tr>
+    <td>CI 1124</td>
+    <td>Global Stories of Education: Literature for Young Adults</td>
+    <td>Global Perspectives & Literature</td>
+  </tr>
+  <tr>
+    <td>ECON 1101</td>
+    <td>Principles of Microeconomics</td>
+    <td>Global Perspectives & Social Sciences</td>
+  </tr>
+  <tr>
+    <td>BBE 1002</td>
+    <td>Biorenewable Resources</td>
+    <td>Technology and Society</td>
+  </tr>
+  <!-- <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr> -->
+  <!-- <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr> -->
+</table>
+
+<!-- TODO -->
+<!-- _持续更新中...有推荐可直接联系---(Decide a person to contact or form to submit_ -->
```

Comment:
This comment will not be shown on the webpage, please confirm it's intended to keep them the current shape

Reply:
I'm wondering should I just leave a wechat ID for people to contact us directly or create a google form for people to submit their suggested course.